test/end-to-end/vouchers/vouchers.spec.js

```diff
@@ -6,12 +6,12 @@ const GridRow = require('../shared/GridRow');
 const components = require('../shared/components');
 
 describe('Voucher Registry', () => {
-  const NUM_VOUCHERS = 28;
+  const NUM_VOUCHERS = 23;
```

Comment:
We should probably figure out a better way to test the number of records downloaded in the future.